src/gui/accountstate.cpp

```diff
@@ -44,6 +45,7 @@ AccountState::AccountState(AccountPtr account)
     , _waitingForNewCredentials(false)
     , _maintenanceToConnectedDelay(60000 + (qrand() % (4 * 60000))) // 1-5min delay
     , _remoteWipe(new RemoteWipe(_account))
+    , _userStatus(new UserStatus(this))
```

Comment:
This is a missing a parent, so I'd say that either you merge parent and accountState parameters in the UserStatus ctor or you pass `this` twice in this call.